init-tools.sh

```diff
@@ -65,55 +61,59 @@ execute_with_retry() {
     return 0
 }
 
+__PKG_ARCH=x64
+if [ "$(uname -m)" == "aarch64" ]; then
+    __PKG_ARCH=arm64
+fi
```

Comment:
I haven't had a functioning arm64 cli to test this with, but it is at least conceptually right.